tools/ci_build/amd_hipify.py

```diff
@@ -144,6 +144,8 @@
                 'tensor/resize_impl.h',
                 'tensor/transpose.cc',
                 'tensor/transpose.h',
+                'tensor/space_depth_ops.cc',
+                'tensor/space_depth_ops.h',
```

Comment:
Can you add the code below to this file line #227? Then the kernels for ROCm can be generated from CUDA implementation.

s = s.replace('cublasHandle', 'RocblasHandle')
s = s.replace('cublas_handle', 'rocblas_handle')
s = s.replace('cublasHandle_t', 'rocblas_handle')

Reply:
Ok. So, we can remove the ROCM implementation files all together ?

Reply:
yes.

Reply:
Done